UNITTESTS/CMakeLists.txt

```diff
@@ -102,62 +67,62 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUNITTEST")
-  "${PROJECT_SOURCE_DIR}/target_h/drivers"
-  "${PROJECT_SOURCE_DIR}/target_h/rtos/include"
-  "${PROJECT_SOURCE_DIR}/stubs"
-  "${PROJECT_SOURCE_DIR}/.."
-  "${PROJECT_SOURCE_DIR}/../features"
-  "${PROJECT_SOURCE_DIR}/../platform/include"
-  "${PROJECT_SOURCE_DIR}/../platform/include/platform"
-  "${PROJECT_SOURCE_DIR}/../platform/mbed-trace/include"
-  "${PROJECT_SOURCE_DIR}/../storage/filesystem/littlefs/include"
-  "${PROJECT_SOURCE_DIR}/../storage/filesystem/fat/include"
-  "${PROJECT_SOURCE_DIR}/../storage/blockdevice/include"
-  "${PROJECT_SOURCE_DIR}/../storage/filesystem/include"
-  "${PROJECT_SOURCE_DIR}/../storage/kvstore/include"
-  "${PROJECT_SOURCE_DIR}/../storage/kvstore/kv_config"
-  "${PROJECT_SOURCE_DIR}/../storage/kvstore/kv_config/include"
-  "${PROJECT_SOURCE_DIR}/../storage/kvstore/tdbstore/include"
-  "${PROJECT_SOURCE_DIR}/../storage/kvstore/filesystemstore/include"
-  "${PROJECT_SOURCE_DIR}/../storage/kvstore/kvstore_global_api/include"
-  "${PROJECT_SOURCE_DIR}/../drivers"
-  "${PROJECT_SOURCE_DIR}/../drivers/include"
-  "${PROJECT_SOURCE_DIR}/../drivers/include/drivers"
-  "${PROJECT_SOURCE_DIR}/../drivers/include/drivers/internal"
-  "${PROJECT_SOURCE_DIR}/../hal"
-  "${PROJECT_SOURCE_DIR}/../hal/include"
-  "${PROJECT_SOURCE_DIR}/../events/include"
-  "${PROJECT_SOURCE_DIR}/../events/include/events/internal"
-  "${PROJECT_SOURCE_DIR}/../events/source"
-  "${PROJECT_SOURCE_DIR}/../rtos/include"
-  "${PROJECT_SOURCE_DIR}/../features/frameworks"
-  "${PROJECT_SOURCE_DIR}/../connectivity/libraries/nanostack-libservice"
-  "${PROJECT_SOURCE_DIR}/../connectivity/libraries/nanostack-libservice/mbed-client-libservice"
-  "${PROJECT_SOURCE_DIR}/../connectivity/netsocket/include"
-  "${PROJECT_SOURCE_DIR}/../features/filesystem/fat"
-  "${PROJECT_SOURCE_DIR}/../features/filesystem/fat/ChaN"
-  "${PROJECT_SOURCE_DIR}/../features/filesystem/bd"
-  "${PROJECT_SOURCE_DIR}/../features/filesystem/"
-  "${PROJECT_SOURCE_DIR}/../features/filesystem/littlefs"
-  "${PROJECT_SOURCE_DIR}/../features/filesystem/littlefs/littlefs"
-  "${PROJECT_SOURCE_DIR}/../connectivity/cellular/include/cellular/framework/API"
-  "${PROJECT_SOURCE_DIR}/../connectivity/cellular/include/cellular/framework/AT"
-  "${PROJECT_SOURCE_DIR}/../connectivity/cellular/include/cellular/framework/device"
-  "${PROJECT_SOURCE_DIR}/../connectivity/cellular/include/cellular/framework"
-  "${PROJECT_SOURCE_DIR}/../connectivity/cellular/include/cellular/framework/common"
-  "${PROJECT_SOURCE_DIR}/../connectivity"
-  "${PROJECT_SOURCE_DIR}/../connectivity/lorawan/include/lorawan"
-  "${PROJECT_SOURCE_DIR}/../connectivity/lorawan/lorastack"
-  "${PROJECT_SOURCE_DIR}/../connectivity/lorawan/lorastack/mac"
-  "${PROJECT_SOURCE_DIR}/../connectivity/lorawan/lorastack/phy"
-  "${PROJECT_SOURCE_DIR}/../connectivity/lorawan"
-  "${PROJECT_SOURCE_DIR}/../connectivity/mbedtls"
-  "${PROJECT_SOURCE_DIR}/../connectivity/mbedtls/include"
+  "${PROJECT_SOURCE_DIR}/UNITTESTS/target_h"
```

Comment:
wouldn't this work with relative paths as we use in the Mbed OS as well?

`./target_h` ?